packages/pools/src/interface.ts

```diff
@@ -16,11 +16,13 @@ export interface Pool {
     amount: Int;
     weight: Int;
   }[];
+  get poolAssetDenoms(): string[];
```

Comment:
IMO, we don't need this getter. It's easy enough to just `map(asset => asset.coinDenom)`